Calibration/LumiAlCaRecoProducers/python/alcaRawPCCProducer_cfi.py

```diff
@@ -1,15 +1,17 @@
44581124,344582148,344585220,344586244,344598532,344602628,344605700,344606724,344609796,344610820,344613892,344617988,344627204,344630276,344631300,344638468,344642564,344646660,344647684,344650756,344659972,344664068,344667140,344668164,344672260,344676356,344679428,344680452,344687620,344688644,344729604,344733700,344736772,344740868,344741892,344744964,344749060,344750084,344753156,344757252,344758276,344761348,344762372,344765444,344766468,344769540,344774660,344777732,344778756,344781828,344782852,344785924,344786948,344790020,344791044,344794116,344795140,344798212,344799236,344802308,344803332,344806404,344807428,344811524,344826884,344827908,344830980,344843268,344844292,344847364,344848388,344851460,344855556,344863748,344867844,344871940,344872964,344876036,344877060,344881156,344892420,344896516,344900612,344901636,344904708,344913924,344921092,344922116,344930308,344933380,344946692,344950788,352588804,352589828,352592900,352593924,352596996,352601092,352602116,352605188,352606212,352609284,352610308,352617476,352621572,352622596,352625668,352626692,352629764,352630788,352633860,352634884,352637956,352638980,352642052,352643076,352646148,352654340,352658436,352659460,352662532,352663556,352666628,352667652,352670724,352671748,352675844,352678916,352679940,352683012,352684036,352687108,352688132,352691204,352692228,352695300,352696324,352699396,352700420,352703492,352704516,352707588,352708612,352711684,352712708,352715780,352719876,352740356,352741380,352744452,352745476,352748548,352749572,352753668,352757764,352761860,352765956,352770052,352774148,352781316,352782340,352786436,352789508,352790532,352793604,352794628,352797700,352798724,352801796,352802820,352805892,352806916,352809988,352811012,352814084,352815108,352850948,352851972,352855044,352856068,352859140,352860164,352863236,352864260,352868356,352871428,352872452,352875524,352876548,352879620,352883716,352892932,352896004,352900100,352901124,352904196,352905220,352908292,352912388,352913412,352920580,352924676,352925700,352928772,352936964,352937988,352941060,352945156,352949252,352950276,352953348,352954372,352957444,352958468,352962564,352966660,352974852,352995332,352999428,353002500,353010692,353015812,353018884,353022980,353024004,353044484,353047556,353048580,353052676,353063940,353068036,353073156,353076228,353118212,353122308,353126404,353129476,353130500,353134596,353141764,353142788,353146884,353149956,353150980,353154052,353155076,353158148,353162244,353163268,353166340,353167364,353170436,353174532,353175556,353178628,353179652,353183748,353187844,353190916,353191940,353195012,353196036,353199108,353200132,353203204,353204228,353207300,353208324,353211396,353216516,353219588,353223684,353224708,353228804,353231876,353235972,353240068,353244164,353249284,353252356,353256452,353268740,353281028,353285124,353286148,353290244,353293316,353298436,353302532,353305604,353313796,353314820,353317892,353321988,353326084,353330180,353331204,353334276,353338372,353339396])
```

Comment:
this is also not very pretty, maybe for a new PR tho, these are the bad components of the Pixel detector right? Maybe one could take the SiPixelQuality object from the PCL from the Pixel team and get this list instead of hardcoding. These things do change in time too, for example L1 is now all good, so the detID from this should not be here (maybe it's already not?)

Reply:
just to chime in @tvami indeed that's the plan to automatize it but afaik for the moment we can leave it hardcoded i'll let @benitezj verify, thx all!

Reply:
Yes, as I wrote above "maybe for a new PR". So I agree that we can leave it hardcoded for now. Thanks @gkrintir for confirming the intent to automatize this.

Reply:
as Georgios points out, the module veto is part a of separate development for future. For now we still use static veto.